node/generate-all-dockerfile.sh

```diff
@@ -87,7 +87,7 @@ for device in $devices; do
 
 		# For armv7hf and armv6hf, if node version is less than 4.x.x (0.10.x 0.12.x) then that image will use binaries from resin, otherwise it will use binaries from official distribution.
```

Comment:
Can you change this comment to reflect the code? i.e if node version is **greater or equal than 4.x.x**...